app/models/subject.rb

```diff
@@ -3,4 +3,132 @@ class Subject < ApplicationRecord
   has_many :courses, through: :course_subjects
 
   validates :code, uniqueness: true
+
+  SUBJECT_CODE_MAPPINGS = {
```

Comment:
I'm kind of torn on whether these constants here or within the exports. Stuff like EBACC and secondary subjects definitely makes sense living here as they're used in other external reporting.

 I'm not sure whether mappings that are only used for this though like this should live here (like this and the status mapping). They should probably live in the export itself. Wdyt?

Reply:
yeah agree, makes sense since the mapping is only related to the export

Reply:
have shifted!